.envrc

```diff
@@ -5,3 +5,4 @@ use flake
 PATH_add .foundry/bin
 PATH_add .cargo/bin
 PATH_add target/debug/
+PATH_add target/release/
```

Comment:
Dunno if this is the best approach. The release build is built with a lower frequency, so might override the debug, or is it the other way around?

Reply:
Ok, I will remove it then.